apps/ml/site/analysis/view-model.jag

```diff
@@ -339,6 +339,15 @@ $( document ).ready(function() {
           }
     });
 
+    if( fromCompare == 'true'){
+         $("#nav-analysis-anchor").attr('href','../project/compare.jag?projectId=' + projectId + '&projectName=' + projectName + '&dataseId=' + datasetId);
+    }
+    else{
```

Comment:
Formatting issue. "else" statement should start right after the "if" closing curly brackets.